torch_tvm/custom_tvm_ops/topi/topi_init.cc

```diff
@@ -0,0 +1,94 @@
+#include <topi/x86/default.h>
+#include <tvm/build_module.h>
+#include <tvm/relay/attrs/transform.h>
+#include <tvm/relay/op_attr_types.h>
+#include <tvm/relay/expr.h>
+#include <tvm/runtime/packed_func.h>
+#include <topi/generic/injective.h>
+
+#include <custom_tvm_ops/relay/layer_norm_attrs.h>
+
+#include "custom_layer_norm_generic_sched.h"
+#include "custom_layer_norm.h"
+
+namespace tvm {
+
+using tvm::relay::OpPatternKind;
+
+class CustomTOPIOpRegisterer {
```

Comment:
nit: name this `RegisterCustomLayerNorm` and make it a struct, removing the `public` just below

Reply:
Got it.

Reply:
So this becomes customer layer norm specific then. I think Lingyi is going to add his registration logic corresponding to his stuff here as well, in that case `RegisterCustomLayerNorm` name would not make sense.

Reply:
that's the point of static initialization -- we avoid blockers by keeping the registration in separate compilation units rather than putting it all into a single file

Reply:
I see. I think thats fair. But note that this would also mean each op we add has its own initialization file, which I think is ok but something to keep in mind. I will rename this to custom_layer_norm_init.cc then.